### DISCUSSION TITLE
My eyes hurts with the new UI, how do i revert to normal?

### DISCUSSION THREAD

**Discussion by snwefly** [community/community#139145] [Other Feature Feedback, Questions, & Ideas]
Old UI, also in incognito mode

![image](https://github.com/user-attachments/assets/53db231d-cc61-431d-9ea7-975f1ecf5e1d)

New UI when logged in, font is disorienting and ui is too white and sharp, its hurts my eyes.
![image](https://github.com/user-attachments/assets/8326b8ce-8e5a-4d6e-b892-b96f52ee84a2)

I already tried every theme from appearance section still no success so far , 

How do i revert to the old one? 

Thanks

**Comment by sam-math** [comment]
Check your browser extensions. One of them might be overwritting your github theme preferences.

**Reply by snwefly** [reply]
I tired on clean chrome install without any extensions but theme only changes when logged in. definitely UI changes is related to the github account.

**Comment by Gogo200202** [comment]
Install [Stylish](https://addons.mozilla.org/en-US/firefox/addon/stylish/). Make custom CSS file.
